packages/react-server-webpack-plugin/src/react-server-webpack-plugin.ts

```diff
@@ -76,39 +79,28 @@ function serverSource(options: Options) {
     ? JSON.stringify(assetPrefix)
     : 'process.env.CDN_URL || "localhost:8080/assets/webpack"';
 
+  const initial = readFileSync(resolve(__dirname, './templates/server.ts'));
+  const source = transformSync(
+    initial.toString(),
+    fillCallExpressionOptions('createServer', {
+      port: serverPort,
+      ip: serverIp,
+      assetPrefix: serverAssetPrefix,
+    }),
+  );
+
   return `
     ${HEADER}
```

Comment:
Could we just include the header as an interpolation in the template as well?

Edit: I see we aren't doing templates quite how I expected, interesting

Reply:
We could pass the header in yeah, but I didn't really see much value in it vs just popping it on the top like this.